### PR TITLE
Update 2023 diversity

### PR DESCRIPTION
**Pull request by metalmatze**
As per CNCF request with updated links.

### CODE REVIEW

File: content/2023-berlin/diversity.md [hunk 1]
```diff
@@ -4,22 +4,36 @@ title: Diversity Scholarships
...
 
-Receipt of Travel Funding does not guarantee entry to the event.  Recipients need to adhere to The Linux Foundation COVID-19 rules and regulations. If the recipient does not meet COVID-19 requirements for the event, the travel fund allotted is null and void. Should the recipient not be able to travel due to COVID-19 and travel has already been booked, the recipient will need to work with airlines and hotels to receive a refund. If a refund is not a possibility, The Linux Foundation will reimburse for incurred travel expenses within the allotted amount.
+## Need-Based
 
-For questions, please contact <a href="mailto:travelfund@linuxfoundation.org">travelfund@linuxfoundation.org</a>.
+Applicants should be active community members who are not being assisted or sponsored by a company or organization, and are unable to attend for financial reasons.
+
+## Maintainer
+
+Applicants should be active Prometheus maintainers who are not being assisted or sponsored by a company or organization, and are unable to attend for financial reasons.
```
Comment (RichiH) [inline content/2023-berlin/diversity.md:19]:
```suggestion
Applicants need to be active Prometheus maintainers who are not being assisted or sponsored by a company or organization, and are unable to attend for financial reasons.
```

**Review by RichiH — COMMENTED** [review]
Historically, we have given free tickets to all Prometheus team members, not tied to need.

I am fine either way, just wanted to make sure this is a deliberate decision, not an oversight.

**Comment by metalmatze** [comment]
Thanks for the suggestions. 
I'll clarify the team members free tickets. It was definitely an oversight for now as I also just read the text copying it from the email.

**Comment by metalmatze** [comment]
Maybe let's get this in for now and figure out the Maintainer stuff for us afterward?
This is mostly for the others anyway.

**Comment by roidelapluie** [comment]
LGTM

**Comment by metalmatze** [comment]
Can someone please merge this for now? CNCF is getting requests with the old information.

**Comment by roidelapluie** [comment]
Thanks

**Pull request merged**